cmd/sysl/cmd_diagram.go

```diff
@@ -50,6 +54,8 @@ func callDiagramGenerator(m *sysl.Module, p *diagramCmd) (string, error) {
 		return sequencediagram.GenerateSequenceDiagram(m, res[0], res[1])
 	case p.EndpointAnalysis:
 		return endpointanalysisdiagram.GenerateEndpointAnalysisDiagram(m)
+	case p.DataDiagram:
```

Comment:
it looks like a user might be able to put in multiple flags, assuming that multiple diagrams can be generated, but it actually only executes one flag (based on the ordering of the switch statement)

Maybe this should use subcommands https://github.com/alecthomas/kingpin#sub-commands so that it's  less confusing to the user, rather than flags?

Reply:
Needs refactoring in the cmd_runner.

Reply:
https://github.com/anz-bank/sysl/issues/857